compiler/ccgtypes.nim

```diff
@@ -54,6 +54,9 @@ proc mangleParamName(m: BModule; s: PSym): Rope =
   ## we cannot use 'sigConflicts' here since we have a BModule, not a BProc.
   ## Fortunately C's scoping rules are sane enough so that that doesn't
   ## cause any trouble.
+  if s.name.s == "_":
```

Comment:
can there be something similar for js? or what prevents this?

Reply:
It just didn't work for JS VM I guess a known bugs

Reply:
tunderscore.nim(9, 27) Error: cannot evaluate at compile time: x

Reply:
in this case we should leave #13443 open (with a note) until we can address this :)

Reply:
why, that's the bug of js vm, not related to my PR?

Reply:
fine, I'll just open a new bug for js+vm then

Reply:
This shouldn't be handled in the backend IMO.

Reply:
but what do you suggest concretely, then?

Reply:
Can't we just mangle `if sfGensym in s.flags`?